src/sortByProps.js

```diff
@@ -0,0 +1,37 @@
+import { reduce, either, curry, prop, sort, comparator, map, lt } from 'ramda';
+
+import isNotArray from './isNotArray';
+
+/**
+ * Sort an list of objects by a list of props (if first prop value is equivalent, sort by second, etc).
+ *
+ * @func sortByProps
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.26.0|v2.26.0}
+ * @category List
+ * @sig [{k: v}] -> [String] -> [{k: v}]
+ * @param {[{k: v}]} list A list of objects to be sorted
```

Comment:
These are [jsdoc](https://jsdoc.app/) params and return. They don't share the same rules as [@sig](https://github.com/ramda/ramda/wiki/Type-Signatures)